starter_scripts/exaslct_within_docker_container.sh

```diff
@@ -52,17 +48,40 @@ if [[ -n "$chown_directories" ]]; then
   chown_directories_cmd="chown -R $(id -u):$(id -g) $chown_directories;"
 fi
 
+BASH_MAJOR_VERSION=$(echo "${BASH_VERSION}" | cut -f1 -d".")
+BASH_MINOR_VERSION=$(echo "${BASH_VERSION}" | cut -f2 -d".")
+
 #For all mount pounts (directories in argument list) we need
 # 1. For the host argument: Resolve relative paths and resolve symbolic links
 # 2. For the container argument: Resolve relative paths, but keep symbolic links
-mount_point_parameter=''
-for mount_point in "${mount_point_paths[@]}"; do
-  if [[ -n "${mount_point}" ]]; then
-    host_dir_name=$(readlink -f "${mount_point}")
-    container_dir_name=$(realpath -s "${mount_point}")
-    mount_point_parameter="$mount_point_parameter-v ${host_dir_name}:${container_dir_name} "
-  fi
-done
+if [[ $BASH_MAJOR_VERSION -lt 5 ]] && [[ $BASH_MINOR_VERSION -lt 4 ]]; then
```

Comment:
The check would fail for bash 3.5 for example. But I think there is no Linux around with such an old version.

Reply:
One possible solution:
```suggestion
function create_version_int() {
    major_version=$(($1<<32))
    minor_version=$2
    echo $((major_version+minor_version))
}
min_bash_version=$create_version_int 4 4
current_bash_version=$create_version_int $BASH_MAJOR_VERSION $BASH_MINOR_VERSION
if [[ current_bash_version -lt min_bash_version ]]; then
```

Reply:
this is not an issue anymore, because this script doesn't work with bash version before 4.0, because before that associative arrays weren't support which are used in mount_point_parsing.sh. Instead, I added a check to switch to slim version, such that the slim version is used not only for MacOSX, but also for bash version before 4.0.
https://tldp.org/LDP/abs/html/bashver4.html